maro/simulator/scenarios/vm_scheduling/topologies/azure.2019.toy/config.yml

```diff
@@ -0,0 +1,69 @@
+# Initial buffer time budget.
+BUFFER_TIME_BUDGET: 0
```

Comment:
refer to _tests/citi_bike/test_bike_scenario.py_
and _tests/data/citi_bike/case_1_

move the test data and config to under test/ folder

Reply:
reminder